cel/folding.go

```diff
@@ -139,6 +145,32 @@ func tryFold(ctx *OptimizerContext, a *ast.AST, expr ast.Expr) error {
 	return nil
 }
 
+func getFunctionOverloadDecls(function *decls.FunctionDecl, overloadIds []string) []*decls.OverloadDecl {
+	var overloads []*decls.OverloadDecl
+	if function == nil {
+		return overloads
+	}
+	for _, o := range function.OverloadDecls() {
+		for _, id := range overloadIds {
+			if id == o.ID() {
+				overloads = append(overloads, o)
+			}
+		}
+	}
```

Comment:
Could you add a function to select an overload by name from the `decl.FunctionDecl`? I think that will simplify some of this code and be generally useful.

Reply:
I instead went with the approach you suggested below of adding `HasLateBinding()` to `decls.FunctionDecl`.